keyboards/binepad/bn003/rules.mk

```
@@ -23,11 +16,7 @@ COMMAND_ENABLE = yes        # Commands for debug and configuration
 SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
 # if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
 NKRO_ENABLE = yes            # USB Nkey Rollover
```

Tiny bit of formatting, but these things irk me.

```suggestion
NKRO_ENABLE = yes           # USB Nkey Rollover
```